imports/test-utils/helpers/waitUntilLoaded.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name waitUntilDataLoaded
+ * @summary Promise function that resolves when an enzyme-mounted React component's prop in question is truthy.
+ *  Useful for waiting until a GraphQL HOC has finished loading data (via Apollo's MockedProvider) in unit tests.
+ * @namespace TestHelpers
+ * @param {ReactWrapper} reactWrapper - result of calling enzyme's mount()
+ * @param {String} componentName - name of component to check
+ * @param {String} propName - name of prop on component to check for truthiness
+ * @returns {Promise} that resolves when prop is truthy
+ */
+export default function waitUntilDataLoaded(reactWrapper, componentName, propName) {
```

Comment:
@aldeed I created this helper function for waiting until a GraphQL HOC has finished loading with the new `isLoading...` props I added. I still have to use a timeout, otherwise the function just keeps calling itself and `MockedProvider` never gets to update loading state to `false`.

Reply:
Thats cool. At least it won't fail if it's not done loading after the first tick.